odxtools/communicationparameter.py

```diff
@@ -47,7 +47,7 @@ def _read_complex_value_from_odx(et_element):
 
 
 def read_communication_param_ref_from_odx(et_element):
-    id_ref = make_ref(et_element)
+    id_ref = et_element.get("ID-REF")
```

Comment:
Please double check in your pdx if the element have a `DOCREF` attribute, if it does, then the ID-REF is not pointing to the same file.